### ISSUE TITLE
Adding new trackers from UE Marketplace pages seems to be broken. 

### ISSUE THREAD

**Issue by jakemelt** [CGeorges/unrealsales-tracker#90]
Seem like Epic changed web page metadata / API access?  

I have noticed that Embed.ly support suddenly stopped working this past week as well.

Any idea what's happening? Can't track down any information on Marketplace API. I would guess they are preparing to shift things to FAB?

**Comment by thsxthz** [comment]
Yes I'm notincing this as well.

**Comment by Rick86** [comment]
Me too :(

**Comment by OtakuMega** [comment]
Me too

**Comment by Ben-Cykyria** [comment]
Me too

**Comment by ThinVeil** [comment]
Just discovered this marketplace tracker a few days ago. I am running in this issue as well. However, does this tool still have active developers? This is a pretty major bug and there is no response to this issue.

**Comment by OtakuMega** [comment]
I just read readme, it will not be updated. I advise you https://orbital-market.com/ it is not so good but it works.

The api of the marketplace works, so I'll have to set up something more amateurs :P

[api example](https://www.unrealengine.com/marketplace/api/assets/asset/50-female-animations)

**Comment by Rick86** [comment]
> I just read readme, it will not be updated. I advise you https://orbital-market.com/ it is not so good but it works.

But Orbital can not save any assets to lists... I now use the fav system from the marketplace again, seems at least they removed the ridiculous 50 items limit, not sure if there is still a limit.

**Comment by SkullShifter** [comment]
> I just read readme, it will not be updated. I advise you https://orbital-market.com/ it is not so good but it works.
> 
> The api of the marketplace works, so I'll have to set up something more amateurs :P
> 
> [api example](https://www.unrealengine.com/marketplace/api/assets/asset/50-female-animations)

Have you ever found any documenation on the Marketplace API? I am very curious to look into it further as well.

**Comment by SkullShifter** [comment]
I guess maybe the issue might be the server unrealsales-tracker is hosted on?  unrealsales.io

I am gonna set up a local test, maybe we can take over the codebase and figure out how to get it working  solidly again

I do know that Epic seems to have changed the metadata structure on the page, so that automartic embeds no longer work with Miro or Embed.ly (both of which I used previously to build my own databases by pasting URLs)

**Comment by OtakuMega** [comment]
@SkullShifter  It's a good idea, I have no idea about React so I had given up xD

The documentation page (github) was removed earlier this year to relocate it but they haven't published it yet. https://github.com/EpicGames

I recommend you extract info from orbital-market [[here](https://github.com/johndpope/OrbitalMarket/blob/1861b5b70e52a1ec78e761064eacba17c6d34960/backend/src/scrapper/unreal/api/index.ts)].

**Comment by OtakuMega** [comment]
> > I just read readme, it will not be updated. I advise you https://orbital-market.com/ it is not so good but it works.
> 
> But Orbital can not save any assets to lists... I now use the fav system from the marketplace again, seems at least they removed the ridiculous 50 items limit, not sure if there is still a limit.

it has a favorites system, it works without registration (it is saved in cookies), but you can't create personalized lists nor does it notify you by mail :(

pda. the new limit of marketplace favorites is 200

![favvvv](https://github.com/VoDoo-Studios/unrealsales-tracker/assets/47985648/8a518854-66fd-44bb-8920-cfaa7f4d71b6)

**Comment by UncleFred121** [comment]
The orbits marketplace doesn't let you filter your list by sales either. Maybe we can run this locally? I'd happily pay someone to get that set up.

**Comment by accidental-expert** [comment]
yeah id be willing to contribute to a bounty to get this working again.

**Comment by UncleFred** [comment]
It looks like the original author is going to discontinue it. He updated his readme a little while ago. I've submitted a request to the Orbital market for an improved favorite list, but so far, no response.

Maybe we could pool our resources and hire someone to make a fork of this? Thoughts on this matt?
